### PR TITLE
Fixes mariadb backups and backups for api-db on alpine

### PR DESCRIPTION
**Pull request by rtprio**
This also requires `MYSQL_USER`  and `MYSQL_PASSWORD` added to the lagoon-env configmap.

Is there any problem with that?

### CODE REVIEW

File: images/mariadb/Dockerfile [hunk 1]
```diff
@@ -27,6 +27,7 @@ do mkdir -p $i; chown mysql $i;  /bin/fix-permissions $i; done
 
 COPY docker-entrypoint.sh /lagoon/entrypoints/90-mariadb-entrypoint
 COPY mysql-backup.sh /lagoon/
+RUN chmod 755 /lagoon/mysql-backup.sh
```
Comment (twardnw) [inline images/mariadb/Dockerfile]: I have found that just setting executeable and committing a script is enough to make it work in the container, eliminating another step from the Dockerfile.

File: images/mariadb/mysql-backup.sh [hunk 2]
```diff
@@ -55,11 +55,11 @@ function delete_old_backups()
...
 
 function mysql_login() {
-  local mysql_login="-u $MYSQL_USER -h $MYSQL_HOST"
+  cmd="-u $MYSQL_USER -h $MYSQL_HOST"
   if [ -n "$MYSQL_PASSWORD" ]; then
-    local mysql_login+=" -p$MYSQL_PASSWORD"
+    cmd="$cmd -p$MYSQL_PASSWORD"
   fi
-  echo $mysql_login
+  echo $cmd
```
Comment (twardnw) [inline images/mariadb/mysql-backup.sh]: might be safer to do this as `` $($cmd) `` or `` `$cmd` ``

Comment (twardnw) [reply images/mariadb/mysql-backup.sh]: nvm, per slack convo it's not useful here 😄

**Review by Schnitzel — APPROVED** [review]
nice!

**Pull request merged**